bin/tictactoe

```diff
@@ -1,3 +1,36 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic Tac Toe!"
+
+    puts " 1 | 2 | 3 "
+    puts "-----------"
+    puts " 4 | 5 | 6 "
+    puts "-----------"
+    puts " 7 | 8 | 9 "
+puts "Select number of players: 0, 1, or 2"
+puts  "To quit press 'q'"
+input = gets.strip
+
+case input
+  when "0"
+    game = Game.new(Computer.new("X"), Computer.new("O"))
+    game.play
+  when "1" 
+    puts "Pick 'X' or 'O'"
+    input = gets.strip.downcase
+    if input == 'x'
+    game =  Game.new(Computer.new("O"),Human.new("X"))
+    game.play
+    elsif input == 'o'
+        game =  Game.new(Computer.new("X"), Human.new("O"))
+      game.play
+    else
+      puts "Invalid input"
+    end
+  when "2"
+    game = Game.new()
+    puts "X goes first"
+    game.play
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.